enderio-machines/src/main/java/com/enderio/machines/common/block/MachineBlocks.java

```diff
@@ -10,25 +10,46 @@
 public class MachineBlocks {
     private static final Registrate REGISTRATE = EIOMachines.registrate();
 
+    public static final BlockEntry<MachineBlock> FLUID_TANK = REGISTRATE
+        .block("fluid_tank", props -> new MachineBlock(props, MachineBlockEntities.FLUID_TANK))
+        .properties(props -> props.strength(2.5f,8))
+        .loot(LootTableUtils::copyNBT)
+        .item()
+        .tab(() -> EIOCreativeTabs.MACHINES)
+        .build()
+        .register();
+
+    public static final BlockEntry<MachineBlock> ENCHANTER = REGISTRATE
+        .block("enchanter", props -> new MachineBlock(props, MachineBlockEntities.ENCHANTER))
+        .properties(props -> props.strength(2.5f,8))
+        .loot(LootTableUtils::copyNBT)
+        .item()
+        .tab(() -> EIOCreativeTabs.MACHINES)
+        .build()
+        .register();
+
     public static final BlockEntry<MachineBlock> SIMPLE_POWERED_FURNACE = REGISTRATE
-        .block("simple_powered_furnace", props -> new MachineBlock(props, MachineBlockEntities.SMELTER))
+        .block("simple_powered_furnace", props -> new MachineBlock(props, MachineBlockEntities.SIMPLE_POWERED_FURNACE))
+        .properties(props -> props.strength(2.5f,8))
+        .loot(LootTableUtils::copyNBT)
         .item()
         .tab(() -> EIOCreativeTabs.MACHINES)
         .build()
         .register();
 
-    public static final BlockEntry<MachineBlock> FLUID_TANK = REGISTRATE
-        .block("fluid_tank", props -> new MachineBlock(props, MachineBlockEntities.FLUID_TANK))
+    public static final BlockEntry<MachineBlock> SIMPLE_ALLOY_SMELTER = REGISTRATE
+        .block("simple_alloy_smelter", props -> new MachineBlock(props, MachineBlockEntities.SIMPLE_ALLOY_SMELTER))
         .properties(props -> props.strength(2.5f,8))
         .loot(LootTableUtils::copyNBT)
         .item()
         .tab(() -> EIOCreativeTabs.MACHINES)
         .build()
         .register();
 
-    public static final BlockEntry<MachineBlock> ENCHANTER = REGISTRATE
-        .block("enchanter", props -> new MachineBlock(props, MachineBlockEntities.ENCHANTER))
+    public static final BlockEntry<MachineBlock> ALLOY_SMELTER = REGISTRATE
+        .block("alloy_smelter", props -> new MachineBlock(props, MachineBlockEntities.ALLOY_SMELTER))
         .properties(props -> props.strength(2.5f,8))
+        .loot(LootTableUtils::copyNBT)
         .item()
         .tab(() -> EIOCreativeTabs.MACHINES)
         .build()
```

Comment:
just as a suggestion: move simple, normal and enhanced MachineBlocks into a group or EnumMap or something, with a helper methods to create those groups. Could unclutter the registrationfiles a bit

Reply:
maybe not for alloy smelter, as it also has to incorporate a furnace, but could be useful for all other machines

Reply:
will do this in a future PR